recipes/imgui/config.yml

```diff
@@ -1,4 +1,8 @@
 versions:
+  "1.92.4b":
```

Comment:
The current failing check comes from the config keys not being the same as thoses set in the `conandata.yml` file, we need to improve the handling for this case